docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
     image: kinto:autograph
 
   megaphone:
-    image: mozilla/megaphone
+    image: mozilla/megaphone:latest
```

Comment:
That change was "pointless" since it doesn't add any value. Right?

Reply:
Trying to dig into this more, it does seem that if a tag is not provided, `default` is assumed.